ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/widgets/editortab/EditorTabWidget.java

```diff
@@ -186,12 +185,6 @@ public void unSelect() {
 
     /** {@inheritDoc} */
     @Override
-    public void setTabPosition(@NotNull TabPosition tabPosition) {
```

Comment:
The feature will be merged to spi branch. Don't pay attention on the policy.

https://github.com/eclipse/che/pull/5812